frontend/workflows/projectCatalog/src/details/quick-links.tsx

```diff
@@ -130,8 +131,48 @@ const QuickLinksCard = ({ linkGroups }: QuickLinksProps) => (
           />
         );
       })}
-    </Grid>
-  </Card>
-);
+    </>
+  );
+};
 
+const QuickLinksCard = ({ linkGroups }: QuickLinksProps) => {
+  const anchorRef = React.useRef(null);
+  const [open, setOpen] = React.useState(false);
+  // Show only the first five quick links, and put the rest in
+  // an overflow popper
+  const firstFive = linkGroups.slice(0, 5);
+  const overflow = linkGroups.slice(5);
+
+  return (
+    <Card>
+      <Grid
+        container
+        item
+        direction="row"
+        alignItems="center"
+        spacing={1}
+        style={{ padding: "10px" }}
+      >
+        <SlicedLinkGroup slicedLinkGroups={firstFive} />
+        {overflow.length > 0 && (
+          <>
+            <IconButton size="small" ref={anchorRef} onClick={() => setOpen(true)}>
```

Comment:
We have a neutral variant for the IconButton, can we try that here?

Reply:
oh yeah oops I had forgotten you added that a while back. thanks yeah that works